codechain/keystore/StorageJson.py

```diff
@@ -48,3 +53,33 @@ def encode(private_key: str, key_type: KeyType, passphrase: str, meta: str):
         "address": address,
         "meta": meta,
     }
+
+
+def decode(json: object, passphrase: str) -> str:
```

Comment:
`json` is definitely an `object` because everything is `object` in Python. Can you narrow the type? `Mapping` or just don't specify would be good. 
`Mapping`: https://docs.python.org/3/library/collections.abc.html#collections.abc.Mapping

Reply:
I removed it.